Userland/Libraries/LibWeb/Layout/FlexFormattingContext.cpp

```diff
@@ -1179,28 +1179,33 @@ void FlexFormattingContext::distribute_any_remaining_free_space()
 
         // 12.2.
         float space_between_items = 0;
-        float space_before_first_item = 0;
+        float initial_offset = 0;
         auto number_of_items = flex_line.items.size();
 
+        int direction = 1;  // LTR
```

Comment:
Seems a little odd to use 1 and -1 for LTR/RTL, can this not just be an enum?

Reply:
That would be cleaner, true

Reply:
done